presidio-analyzer/analyzer/predefined_recognizers/credit_card_recognizer.py

```diff
@@ -51,5 +52,6 @@ def digits_of(n):
             checksum += sum(digits_of(d * 2))
         return checksum % 10
 
+    #  pylint: disable=attribute-defined-outside-init
     def __sanitize_value(self, text):
         self.sanitized_value = text.replace('-', '').replace(' ', '')
```

Comment:
This is kind of bad practice. Either we declare self.sanitized_value in the init or we have the method return a value